### DISCUSSION TITLE
Error: WebSocket connection closed abnormally with code: 3000

### DISCUSSION THREAD

**Discussion by Confucian-e** [rainbow-me/rainbowkit#1471] [Q&A]
I got some trouble connecting to wallet.
I rasied an issue before: https://github.com/wagmi-dev/wagmi/discussions/2827

**Comment by DanielSinclair** [comment]
@Confucian-e Did you supply a valid `projectId`? WalletConnect will block websockets requests when that is valid; you'd just need to create one on WalletConnect Cloud. Alternatively, there are quite a few instances where if you use a VPN or live in a region with a firewall, then WalletConnect websockets will be blocked.
